actors/builtin/miner/policy.go

```diff
@@ -171,8 +171,8 @@ var consensusFaultReporterInitialShare = BigFrac{
 }
 var consensusFaultReporterShareGrowthRate = BigFrac{
 	// PARAM_FINISH
-	numerator:   big.NewInt(101251),
-	denominator: big.NewInt(100000),
+	numerator:   big.NewInt(1004356),
+	denominator: big.NewInt(1000000),
```

Comment:
I gather that this ratio is derived from finality. Please document the arithmetic involved here or, even better, do the calculation of numerator in code if possible.